### ISSUE TITLE
difficulty in running without CUDA

### ISSUE THREAD

**Issue by harryhoch** [YiyeZhangLab/DICE#1]
There seems to be a bug that might impact running without Cuda. 

Starting at line 204,

            if mask_BoolTensor!=None:
                if self.cuda:
                    mask_BoolTensor = mask_BoolTensor.cuda()
                output_c = output_c.masked_fill(mask = mask_BoolTensor, value=torch.tensor(0.0) )

The problem here is that "self.cuda" is not the name of the class atttribute - it's self.para_cuda in this class. Thus, the inner if block should read..

if self.para_cuda:
    ....
